src/main/java/core/basesyntax/FigureProducer.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureProducer {
+
+    public static final int COUNT_OF_FIGURE = 5;
```

Comment:
```suggestion
    private static final int COUNT_OF_FIGURE = 5;
```
if we use some element only once in the class we should keep it as strict as possible